.github/workflows/flutterfmt.yml

```diff
@@ -5,25 +5,24 @@ on:
       - main
   pull_request:
     paths:
-      - '.github/workflows/flutterfmt.yml'
-      - '.github/workflows/flutterfmt.sh'
-      - '**.dart'
+      - ".github/workflows/flutterfmt.yml"
+      - ".github/workflows/flutterfmt.sh"
+      - "**.dart"
```

Comment:
What caused this change of quote style?

Reply:
i have a yaml formatter in vscode, i can revert those changes